iroha-balancer-client/src/main/kotlin/soramitsu/irohautils/balancer/client/service/IrohaBalancerClientService.kt

```diff
@@ -25,19 +24,16 @@ open class IrohaBalancerClientService @JvmOverloads constructor(
     private val objectMapper = ObjectMapper()
 
     private val connection by lazy {
-        if (rmqConfig.username != null && rmqConfig.password != null) {
-            logger.info { "Authenticate RMQ user: ${rmqConfig.username}" }
-            factory.username = rmqConfig.username
-            factory.password = rmqConfig.password
+        if (rmqConfig.username == null || rmqConfig.password == null) {
+            logger.error { "RMQ credentials are not provided" }
+            throw ClientErrorException("RMQ credentials are not provided. Please, check your settings.")
```

Comment:
actually it seems to be okay not to provide credentials since it is not a mandatory part of the rmq deployment